From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ronald Claveau <linux-kernel-dev@aliel.fr>
Date: Fri, 3 Apr 2026 12:01:34 +0200
Subject: dt-bindings: usb: xhci: Add compatible for Amlogic T7 SoC

The Amlogic T7 SoC uses Corigine USB IP that is xHCI compliant.
Add a dedicated compatible to apply the required quirks:
the controller incorrectly reports 64-bit DMA support,
and a reset is needed on resume.

Signed-off-by: Ronald Claveau <linux-kernel-dev@aliel.fr>
---
 Documentation/devicetree/bindings/usb/generic-xhci.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/generic-xhci.yaml b/Documentation/devicetree/bindings/usb/generic-xhci.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/usb/generic-xhci.yaml
+++ b/Documentation/devicetree/bindings/usb/generic-xhci.yaml
@@ -34,6 +34,9 @@ properties:
         enum:
           - brcm,xhci-brcm-v2
           - brcm,bcm7445-xhci
+      - description: Amlogic SoCs with xHCI
+        enum:
+          - amlogic,t7-xhci
       - description: Generic xHCI device
         const: xhci-platform
         deprecated: true
-- 
Armbian

